src/test/java/com/liferay/upgrades/analyzer/exporter/GamePlanProjectDependencyExporterTest.java

```diff
@@ -0,0 +1,52 @@
+package com.liferay.upgrades.analyzer.exporter;
+
+import com.liferay.upgrades.analyzer.project.dependency.exporter.GamePlanProjectDependencyExporter;
+import com.liferay.upgrades.analyzer.project.dependency.graph.builder.ProjectsDependencyGraph;
+import com.liferay.upgrades.analyzer.project.dependency.graph.builder.ProjectsDependencyGraphBuilder;
+import com.liferay.upgrades.analyzer.project.dependency.model.Project;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class GamePlanProjectDependencyExporterTest {
+
+    @Test
+    public void testExportGamePlan() throws IOException {
+        final var expectedMessage = "Game Plan generated at ";
+
+        ProjectsDependencyGraphBuilder projectsDependencyGraphBuilder = new ProjectsDependencyGraphBuilder();
+        GamePlanProjectDependencyExporter gamePlanProjectDependencyExporter = new GamePlanProjectDependencyExporter();
```

Comment:
Could you please declare the `GamePlanProjectDependencyExporter` instance just when it is to be used?
```
GamePlanProjectDependencyExporter gamePlanProjectDependencyExporter = new GamePlanProjectDependencyExporter();
final var result = gamePlanProjectDependencyExporter.export(projectsDependencyGraph);
```
Or if you prefer, you can do this inline:
```
new StartupCsvProjectDependencyExporter().export(projectsDependencyGraph);
```

Reply:
Just to Exporters